MarkdownCombiner.py

```diff
@@ -0,0 +1,114 @@
+# For correcting the markdown output for pandoc / combining them into one file
+# Changes will need to be made for this if the 'level fix' is merged (multiple level 4 csv and how doorstop handles it)
+# For now, doing it like this for it to work as the way it is as changes are still being done to the requirements.
+# Updates to markdown being used and conversion methods will adjust what needs to be fixed for it to work as intended.
+
+# Imports -------------------------------------------------------------------------------------------------------------
+
+import glob
+import os
+
+# Variables -----------------------------------------------------------------------------------------------------------
+
+# File variables
+md_files = sorted(glob.glob("dist/L*.markdown"))
+output_file = "dist/Pearl_Requirements.markdown"
+
+# Dict of items to replace for each line (varies on version / temp fix) for the combined markdown file generation.
+replacements = {"==================": "------------------", "================": "----------------",
+                "==============": "--------------", "=======": "-------", ":*\\": ":*", "\\_": "_",
+                "(L0.markdown#L": "(#L", "(L1.markdown#L": "(#L", "(L2.markdown#L": "(#L", "(L3.markdown#L": "(#L",
+                "(L4-FOA-M1.markdown#L4-FOA-M1": "(#L4-FOA-M1", "(L4-FOA-M2.markdown#L4-FOA-M2": "(#L4-FOA-M2",
```

Comment:
is this something that would get replaced if a config file were in place?

Reply:
This actual is no longer needed in the doorstop update (this script gets archived / temp fix for github links in one document).